src/components/dropdown-menu/DropdownMenuGroupLabel.tsx

```diff
@@ -20,14 +20,42 @@
 
 import styled from '@emotion/styled';
 import * as radixDropdownMenu from '@radix-ui/react-dropdown-menu';
+import { forwardRef, ReactNode } from 'react';
 
 import { cssVar } from '~utils/design-tokens';
 
-export const DropdownMenuGroupLabel = styled(radixDropdownMenu.Label)`
+export interface DropdownMenuGroupLabelProps extends radixDropdownMenu.DropdownMenuLabelProps {
+  /**
+   * Optional content to display at the end of the group label element.
+   */
+  suffix?: ReactNode;
+}
+
+export const DropdownMenuGroupLabel = forwardRef<HTMLDivElement, DropdownMenuGroupLabelProps>(
+  (props, ref) => {
+    const { children, suffix, ...rest } = props;
+
+    return (
+      <StyledDropdownMenuGroupLabel ref={ref} {...rest}>
+        {children}
+
+        {suffix && <span>{suffix}</span>}
```

Comment:
Do we really want to wrap it in a span? Aren't we usually using a wrapper that put stuff in flex in case there are mutliple tags in the suffix?